awx/ui_next/src/screens/Template/shared/WebhookSubForm.jsx

```diff
@@ -85,9 +90,12 @@ function WebhookSubForm({ i18n, enableWebhooks }) {
     useCallback(async () => {
       const {
         data: { webhook_key: key },
-      } = await JobTemplatesAPI.updateWebhookKey(id);
+      } =
+        templateType === 'job_template'
```

Comment:
seems a little weird having await in the ternary like this, I'd probably set the updateWebhookKey in a var above and then just call that here.  in other words

const updateWebhookKey = templateType === 'job_template' ? JobTemplatesAPI.updateWebhookKey : WorkflowJobTemplatesAPI.updateWebhookKey

await updateWebhookKey(id);

Reply:
Good suggestion.